### PR TITLE
LOK-2509: Monitoring Policies - save policy / enabled status

### PR DESCRIPTION
**Pull request by Syed-Bilal-Haider-Dataq**
## Description
Issues with policy enabled status and saving a policy.

Sometimes clicking on Save Policy in the Edit Details panel doesn't do anything, doesn’t seem to send the graphQL request

If a user changes Enabled status in the Edit Details panel, then clicks Save Policy, the list should be refreshed and the enabled status should display correctly in the table, even while the user is still on the Edit Details panel

There should be a Save Policy button on the policy edit screen in Basic Information tab when in Edit mode (not just on the Alert Rules tab). Also when in Edit mode, remove the Next button in Basic Information and replace with this Save Policy button.
## Jira link(s)
- https://opennms.atlassian.net/browse/LOK-2509

## Checklist
* [ ] Follows Lōkahi's [development guidelines.](https://github.com/OpenNMS-Cloud/lokahi/wiki/Development-Guidelines)
* [ ] Appropriate reviewer(s) have been selected.
* [ ] Jira issue(s) have been updated to "In Review".
* [ ] Includes [appropriate tests.](https://github.com/OpenNMS-Cloud/lokahi/wiki/Test-Strategy)
* [ ] Documentation has been updated as necessary.
* [ ] Notify devops of changes to the Charts.
* [ ] Notify documentation team of any changes to names of screens or features (affects URLs).

### CODE REVIEW

File: ui/src/components/MonitoringPolicies/MonitoringPoliciesDetailPanel.vue [hunk 1]
```diff
@@ -109,6 +112,10 @@ const icons = markRaw({
   deleteIcon: Delete
 })
 
+const savePolicyEnableDisable = computed(
```
Comment (synqotik) [inline ui/src/components/MonitoringPolicies/MonitoringPoliciesDetailPanel.vue:115]: Perhaps rename to `isSavePolicyDisabled`.

Also, let's change the name of `enableDisable` to be `isPolicyEnabled`.

Hopefully that will make it a little clearer what these properties do.

**Review by synqotik — CHANGES_REQUESTED** [review]
Looks good. I'm just requesting a couple of name changes to make it a little clearer.

File: ui/src/components/MonitoringPolicies/MonitoringPoliciesEditPanel.vue [hunk 2]
```diff
@@ -41,6 +47,11 @@ const handleNext = () => {
   }
 }
 
+const isPolicyEditable = computed(() => store.policyEditMode === 2)
+const savePolicyEnableDisable = computed(
```
Comment (synqotik) [inline ui/src/components/MonitoringPolicies/MonitoringPoliciesEditPanel.vue:51]: This should also be renamed to `isSavePolicyDisabled`

File: ui/src/components/MonitoringPolicies/MonitoringPoliciesDetailPanel.vue [hunk 3]
```diff
@@ -155,11 +162,10 @@ const updateMonitoringPolicyStatus = (newStatus: boolean) => {
 }
 
 const savePolicy = async () => {
-  if (enableDisable.value) {
-    const result = await store?.savePolicy({status: enableDisable.value})
-    if (result) {
-      router.push('/monitoring-policies-new/')
-    }
+  const result = await store?.savePolicy({status: enableDisable.value})
```
Comment (synqotik) [inline ui/src/components/MonitoringPolicies/MonitoringPoliciesDetailPanel.vue:165]: Please also make this change, as requested:

change the name of `enableDisable` to be `isPolicyEnabled.`

**Review by synqotik — CHANGES_REQUESTED** [review]
A couple of changes.

**Comment by faizanjanjua** [comment]
LGTM!

**Review by faizanjanjua — APPROVED** [review]
LGTM!

**Review by synqotik — APPROVED** [review]

**Pull request merged**